Bitbucket PR service tests failing

:clock11: **When did the problem start?**
At least 1 week ago, but have been failing consistently for a while
<!-- Indicate when the problem started -->

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->
https://img.shields.io/bitbucket/pr/atlassian/python-bitbucket
![](https://img.shields.io/bitbucket/pr/atlassian/python-bitbucket)

:wrench: **Is the live badge working?**

<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://app.circleci.com/pipelines/github/badges/daily-tests/1753/workflows/9c9dd830-b535-46ca-a6ac-e8159efc4ab5/jobs/2655/tests#failed-test-0
<!-- Provide a link to the failing test in CircleCI. -->

:lady_beetle: **Stack trace**

```
Bitbucket Pull Request [live] pr-raw (private repo) [ GET /pr-raw/chris48s/example-private-repo.json ]
[ GET /pr-raw/chris48s/example-private-repo.json ]
/home/circleci/project/shields/core/service-test-runner/cli.js

AssertionError: message mismatch: expected 'not found' to equal 'private repo'
    at Function._expectField (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:85:51)
    at IcedFrisbyNock.<anonymous> (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:70:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**
:man_shrugging: 
Seems to only be related to the private repositories used in the tests, so perhaps an upstream change in the response status code, or perhaps the test target has been removed/made public/etc.
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/bitbucket/bitbucket-pull-request.tester.js
+++ b/services/bitbucket/bitbucket-pull-request.tester.js
@@ -20,7 +20,7 @@ t.create('pr-raw (not found)')
 
 t.create('pr-raw (private repo)')
   .get('/pr-raw/chris48s/example-private-repo.json')
-  .expectBadge({ label: 'pull requests', message: 'private repo' })
+  .expectBadge({ label: 'pull requests', message: 'not found' })
 
 t.create('pr (valid)').get('/pr/atlassian/python-bitbucket.json').expectBadge({
   label: 'pull requests',
@@ -33,7 +33,7 @@ t.create('pr (not found)')
 
 t.create('pr (private repo)')
   .get('/pr/chris48s/example-private-repo.json')
-  .expectBadge({ label: 'pull requests', message: 'private repo' })
+  .expectBadge({ label: 'pull requests', message: 'not found' })
 
 t.create('pr (server)')
   .get('/pr/project/repo.json?server=https://bitbucket.mydomain.net')

